addons/api/addon/serializers/storage-policy.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import ApplicationSerializer from './application';
+
+export default class StoragePolicySerializer extends ApplicationSerializer {
+  // =methods
+
+  /**
+   * @return {object}
+   */
+  serialize() {
```

Comment:
Why do we have these these serialize and normalize functions? Are they just placeholders? I'd prefer to not have them until we actually need them but if we don't need to do extra serializer/normalize logic we should just delete them

Reply:
yes these were placeholders, I'll delete these